src/components/Persona/Persona.scss

```diff
@@ -25,11 +25,16 @@ $ms-color-initials-darkRed: 		#b91d47;
 
 
 .ms-Persona {
-	@include ms-font-m;
-	@include ms-u-normalize;
-	display: table;
-	line-height: 1;
-	position: relative;
+  @include ms-font-m;
```

Comment:
Looks like the indentation has changed on these lines. Could you revert these lines to how they originally were?